package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -620,22 +649,47 @@ def _read_next_timestep(self):
             except IndexError:
                 raise ValueError("No coordinate information detected")
         elif not self.lammps_coordinate_convention in convention_to_col_ix:
-            raise ValueError(f"No coordinates following convention {self.lammps_coordinate_convention} found in timestep")
```

Comment:
No need to change this back from an f-string. In general, avoid changing things that are not directly related to your PR or fix as it can mess up git history a bit.